src/include/daos_obj.h

```diff
@@ -113,6 +114,12 @@ daos_otype_t_is_valid(enum daos_otype_t type)
 	return type <= DAOS_OT_MAX;
 }
 
+static inline bool
+daos_pa_domain_is_valid(uint32_t pa_domain)
```

Comment:
Should be a minor API version bump for adding a new API?

Reply:
@mchaarawi do you think we should add a minor API version bump for this?

Reply:
i don't think this API will see much use, but i agree it's a good practice to bump the minor version when adding new APIs in any case.